addons/website_event_booth_sale/controllers/event_booth.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.website_event.controllers.main import WebsiteEventController
+
+
+class WebsiteEventBoothController(WebsiteEventController):
+
+    @http.route()
+    def event_booth_main(self, event):
+        event = event.with_context(pricelist=request.website.id)
+        if not request.context.get('pricelist'):
+            pricelist = request.website.get_current_pricelist()
+            if pricelist:
+                event = event.with_context(pricelist=pricelist.id)
```

Comment:
Code is strange, you set twice pricelist in context, based on request.context ... Not sure to understand the priority and source of pricelist.